mainapp/templates/login.html

```diff
@@ -21,6 +20,13 @@ <h2>Login</h2>
                         required>
                 </div>
                 <input type="submit" class="btn btn-primary" value="Login">
+                <a>&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;</a>
+                <button class="btn btn-primary" onclick="goBack()">Back</button>
+                <script>
```

Comment:
How about just a link to the index page instead of back button?

Reply:
I tried that but it didn't work. Had discussion with Alex and decided to go with this.